### ISSUE TITLE
How to retrieve original response body in case of 400 errors.

### ISSUE THREAD

**Issue by shivam-kapila** [plamere/spotipy#645]
Is there a way to retrieve the original response body in case of 400 errors. e.g. The error message in case of error #503 is
```
{
    "error": "invalid_grant",
    "error_description": "Refresh token revoked"
}
```
But spotipy only shows 400 Bad Request and not the original message `Refresh token revoked.` A way to access the original JSON would help better debug issues in client apps.

**Comment by Peter-Schorn** [comment]
I just made a pull request that fixed your issue.

**Comment by shivam-kapila** [comment]
@Peter-Schorn thanks for the action :)

**Comment by stephanebruckert** [comment]
Released in 2.18.0
